sdk-lib/src/main/java/cash/z/ecc/android/sdk/Synchronizer.kt

```diff
@@ -175,29 +161,28 @@ interface Synchronizer {
     suspend fun getTransparentAddress(account: Account = Account.DEFAULT): String
 
     /**
-     * Sends zatoshi.
-     *
      * @param usk the unified spending key associated with the notes that will be spent.
      * @param zatoshi the amount of zatoshi to send.
      * @param toAddress the recipient's address.
      * @param memo the optional memo to include as part of the transaction.
-     *
-     * @return a flow of PendingTransaction objects representing changes to the state of the
-     * transaction. Any time the state changes a new instance will be emitted by this flow. This is
-     * useful for updating the UI without needing to poll. Of course, polling is always an option
-     * for any wallet that wants to ignore this return value.
+     * @return The transaction id of the sent transaction.
      */
-    fun sendToAddress(
+    @Throws(TransactionEncoderException::class, TransactionSubmitException::class)
+    suspend fun sendToAddress(
         usk: UnifiedSpendingKey,
         amount: Zatoshi,
         toAddress: String,
         memo: String = ""
-    ): Flow<PendingTransaction>
+    ): Long
 
-    fun shieldFunds(
+    /**
+     * @return The transaction id of the sent transaction.
```

Comment:
I'd clarify this is the internal id and not the "txid" on chain to make it really obvious, because this number can change if the chain re-orgs so it shouldn't be stored long term